generator/graphql-kotlin-federation/src/test/kotlin/com/expediagroup/graphql/generator/federation/validation/integration/FederatedKeyDirectiveIT.kt

```diff
@@ -104,15 +104,6 @@ class FederatedKeyDirectiveIT {
         assertTrue(validatedType.hasAppliedDirective("key"))
     }
 
-    @Test
-    fun `@extended type should specify @key directive`() {
```

Comment:
this test is no longer valid as we are not scanning for `@extends` directive so the underlying type won't be picked up anymore